trytravis.py

```diff
@@ -31,8 +31,8 @@
 __author__ = 'Seth Michael Larson'
 __email__ = 'sethmichaellarson@protonmail.com'
 __license__ = 'Apache-2.0'
-__url__ = 'https://github.com/SethMichaelLarson/trytravis'
-__version__ = '1.0.4'
+__version__ = '3'
```

Comment:
Version should be `2.0` if we follow semver.

Reply:
ok